src/main/java/io/codelex/test/CreditCard.java

```diff
@@ -0,0 +1,26 @@
+package io.codelex.test;
+
+class CreditCard extends Card {
+    public CreditCard(String number, String owner, String ccv, double balance) {
+        super(number, owner, ccv, balance);
+    }
+
+    @Override
+    public void addMoney(double amount) throws NotEnoughFundsException {
```

Comment:
Ā nu pamanīju vēl šeit - nevajag te to throws NotEnoughFundsException - jo addMoney metode to nevar mest (izlabo attiecīgi arī Card klasi un DebitCard)